web/src/main/webapp/resources/js/atsy-new-application-popup.js

```diff
@@ -0,0 +1,46 @@
+var positionDefaultValueInEng = 'Please choose a position';
+var applicationSourceDefaultValueInEng = 'Please choose an application source';
+var positionDefaultValueInHun = 'Kérem válasszon egy pozíciót';
+var applicationSourceDefaultValueInHun = 'Kérem válasszon egy jelentkezési forrást';
+
+function validateApplicationSourceAndPositionDropDowns() {
+    var positionIsValid = positionValidator();
+    var applicationSourceIsValid = applicationSourceValidator();
+    return (positionIsValid && applicationSourceIsValid);
+}
+
+function positionValidator() {
+    var ddl = document.getElementById('position');
+    var selectedValue = ddl.options[ddl.selectedIndex].value;
+
+    if (selectedValue == positionDefaultValueInEng) {
+        document.getElementById('position_error').innerHTML = positionDefaultValueInEng;
+        return false;
+    }
+
+    if (selectedValue == positionDefaultValueInHun) {
+        document.getElementById('position_error').innerHTML = positionDefaultValueInHun;
```

Comment:
You can initialize your common variables in `$(document).ready()` and use them where needed. This way you don't query the whole DOM again and again only once when document is loaded.